src/main/java/org/jenkins/plugins/lockableresources/LockStepExecution.java

```diff
@@ -1,6 +1,5 @@
 package org.jenkins.plugins.lockableresources;
 
-import java.io.IOException;
```

Comment:
Try

    git checkout master -- src/main/java/org/jenkins/plugins/lockableresources/LockStepExecution.java

to revert irrelevant changes.